### ISSUE TITLE
Getting started section out of date

### ISSUE THREAD

**Issue by nklb** [gonum/website#29]
I'm new to go, thought gonum was interesting and wanted to try it. I followed the instructions on the intro website on how to get started and how to download the libraries.
The first command `go get -u -t gonum.org/v1/gonum/...` seemed to download something but I obtained this deprecation message:

```
go get: installing executables with 'go get' in module mode is deprecated.
        Use 'go install pkg@version' instead.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
```

So I'm not sure if it was installed correctly. Tried a bit if I could find the correct `go install` command but didn't succeed. The second command to run the tests `go test gonum.org/v1/...` failed with `go: warning: "gonum.org/v1/..." matched no packages`.
-->
### What is the URL of the page with the issue?
https://www.gonum.org/post/intro_to_gonum/

### Additional information.
I run macOS and installed go using mac ports.
`go version go1.17.7 darwin/amd64`

**Comment by kortschak** [comment]
The situation with installation is simpler now with the advent of modules, though has become more complicated to describe. Strictly speaking, it's not necessary to install Gonum because the built tool will get it for you when you build things. When you are in a module you are working on, `go get gonum.org/v1/gonum` within the module will add it to the go.mod file and get the code and place it in the pkg cache.

**Comment by kortschak** [comment]
Closed by #30.
